db_link/zu_lib_sql_link.php

```diff
@@ -1,9 +1,10 @@
 <?php
 
-// passwords are in this separate file for easy distribution of code
+// Database configuration from environment variables
+define('SQL_DB_HOST', getenv('DB_HOST') ?: 'db');
```

Comment:
Why 'db' ? Seems to me that this crashes the non docker install.

Reply:
Yes, I changed it already to "define('SQL_DB_HOST', getenv('DB_HOST') ?: 'localhost');" so that it runs fine also without .env file.